### PR TITLE
Add presentation of User-Group 20210311

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: content/pages/98_meetings_de.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Status: hidden
 
 Hier befindet sich die Liste der Meetings der Gruppe. In jeder Seite befinden sich die Entscheidungen / Bemerkungen des Meeting.
 
+### 2021
+
+* [Online, 11.03.2020](meeting_minutes/2020-03-11)
```
Comment (rbovard) [inline content/pages/98_meetings_de.md]:
```suggestion
* [Online, 11.03.2021](meeting_minutes/2021-03-11)
```

File: content/pages/98_meetings_fr.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Status: hidden
 
 Ici se trouve une liste des meetings du groupe. Chaque page regroupe les décisions / remarques prises.
 
+### 2021
+
+* [Online, 11.03.2020](meeting_minutes/2020-03-11)
```
Comment (rbovard) [inline content/pages/98_meetings_fr.md]:
```suggestion
* [Online, 11.03.2021](meeting_minutes/2021-03-11)
```

**Review by rbovard — CHANGES_REQUESTED** [review]
Thanks for the work @pfanguin 

I suggested two littles fixes ;)

@svareg Do you want to add a PV (or text) in the meeting page?

File: content/pages/98_meetings_fr.md (hunk 2 above)
Comment (svareg) [reply content/pages/98_meetings_fr.md]: Hello!

Yes, as soon as it is done... 
I'm a bit late for the PV, I'll try to finish it for tomorrow.
Thanks !

**Comment by pfanguin** [comment]
@svareg : it's done !

**Review by rbovard — APPROVED** [review]
Thanks a lot!

If the links are working (I didn't tested live) you can merge.

**Pull request merged**